Amplify/DevMenu/View/DevMenuList.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SwiftUI
+
+/// View containing a list of developer menu items
+@available(iOS 13.0.0, *)
+struct DevMenuList: View {
+
+    /// Integer tags to identify row items in Dev Menu
+    public static let tagEnvironmentInformation = 0
+    public static let tagDeviceInformation = 1
+    public static let tagViewLogs = 2
+    public static let tagFileIssue = 3
+
+    /// Title corresponding to row items in Dev Menu
+    public static let titleEnvironmentInformation = "View Environment Information"
+    public static let titleDeviceInformation = "View Device Infomation"
+    public static let titleViewLogs = "View Logs"
+    public static let titleFileIssue = "File Issue"
+
+    private let amplifyDevMenuListItems: [DevMenuItem] =
+    [
+        DevMenuItem(title: titleEnvironmentInformation, tag: tagEnvironmentInformation),
+        DevMenuItem(title: titleDeviceInformation, tag: tagDeviceInformation),
+        DevMenuItem(title: titleViewLogs, tag: tagViewLogs),
+        DevMenuItem(title: titleFileIssue, tag: tagFileIssue)
```

Comment:
maybe we could add some icons? :)

SF Symbols offer a bunch of built-in icons: https://developer.apple.com/design/human-interface-guidelines/sf-symbols/overview/ (there's even a native Mac App https://developer.apple.com/sf-symbols/)

You can use them like:

```swift
Image(systemName: "plus.circle.fill")
    .font(.system(size: 24, weight: .thin))
    .foregroundColor(.secondary)
```